src/test/scala/cromwell/engine/backend/CromwellRuntimeAttributeSpec.scala

```diff
@@ -1,72 +1,68 @@
 package cromwell.engine.backend
 
+import java.util.UUID
+
 import com.google.api.services.genomics.model.Disk
 import cromwell.engine.backend.runtimeattributes.{AttributeMap, CromwellRuntimeAttributes, ContinueOnReturnCodeFlag, ContinueOnReturnCodeSet}
 import cromwell.engine.workflow.WorkflowOptions
 import wdl4s.NamespaceWithWorkflow
+import cromwell.CromwellTestkitSpec.TestWorkflowManagerSystem
 import cromwell.engine.backend.CromwellRuntimeAttributeSpec._
+import cromwell.engine.backend.jes.JesBackend
+import cromwell.engine.backend.local.LocalBackend
+import cromwell.engine.backend.runtimeattributes._
+import cromwell.engine.workflow.BackendCallKey
+import cromwell.engine.{WorkflowDescriptor, WorkflowId}
+import cromwell.util.SampleWdl
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.Tables.Table
 import org.scalatest.{EitherValues, FlatSpec, Matchers}
 
 import scala.util.{Failure, Success}
 
 object CromwellRuntimeAttributeSpec {
-  val DockerValuePlaceholder = "DOCKERVALUEPLACEHOLDER"
-  val WdlValueForDockerAttribute = "ubuntu:latest"
-  val WorkflowWithRuntime =
-    """
-      |task ps {
-      |  command {
-      |    ps
-      |  }
-      |  output {
-      |    File procs = stdout()
-      |  }
-      |  runtime {
-      |    docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |task cgrep {
-      |  String pattern
-      |  File in_file
-      |  command {
-      |    grep '${pattern}' ${in_file} | wc -l
-      |  }
-      |  output {
-      |    Int count = read_int(stdout())
-      |  }
-      |  runtime {
-      |    docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |task wc {
-      |  File in_file
-      |  command {
-      |    cat ${in_file} | wc -l
-      |  }
-      |  output {
-      |    Int count = read_int(stdout())
-      |  }
-      |  runtime {
-      |     docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |workflow three_step {
-      |  call ps
-      |  call cgrep {
-      |    input: in_file=ps.procs
-      |  }
-      |  call wc {
-      |    input: in_file=ps.procs
-      |  }
-      |}
-      |
-    """.stripMargin.replace(DockerValuePlaceholder, WdlValueForDockerAttribute)
 
-  val WorkflowWithoutRuntime =
-    """
+  object WorkflowWithStaticRuntime extends SampleWdl {
```

Comment:
I don't have a strong opinion. I'll throw out a few bits of my general philosophy which could apply here though (tl;dr - do whatever you want):

- I don't really care that much about code cleanliness of Spec files
- A file like SampleWdl which is intended to be a bundle of WDLs commonly used in tests I'm totally OK with being large. That said, perhaps some sort of subsetting (e.g. a package of sample WDLs, perhaps with some easy object which collates them or something, who knows)
- If this isn't duplicated it doesn't bother me having one off WDLs in their own tests. To me SampleWdl is mostly useful for reused workflows
- Even if it is duplicated I go back to my first bullet point. As long as this isn't a case where one would want to ensure that changing WDL foo affected everything using that specific WDL it's fine.